app/api/api_v1/schemas/search.py

```diff
@@ -251,6 +251,11 @@ class SearchResponseFamily(BaseModel):
     The geographical location of the family in ISO 3166-1 alpha-3
     """
 
+    family_geographies: List[str]
+    """
+    The geographical locations of the family in ISO 3166-1 alpha-3
```

Comment:
**suggestion:**
```suggestion
    The geographical locations of the families in ISO 3166-1 alpha-3
```

Reply:
So in this context it's we have a search response for a family and thus singular!